src/components/footer/Footer.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
```

Comment:
this component should be a [**stateless functional components**](https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html#stateless-functional-components)